include/wintls/beast.hpp

```diff
@@ -0,0 +1,79 @@
+//
+// Copyright (c) 2023 Kasper Laudrup (laudrup at stacktrace dot dk)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+
+#ifndef BOOST_WINTLS_BEAST_HPP
+#define BOOST_WINTLS_BEAST_HPP
+
+#if __has_include("boost/beast.hpp")
```

Comment:
I would prefer that this conditional was removed and the header was only included when it could actually be used.